### DISCUSSION TITLE
现在加密文件夹只要输一次正确的密码，在不改文件夹密码的情况下就会一直有效

### DISCUSSION THREAD

**Discussion by xiaobudianGG** [AlistGo/alist#576] [Q&A]
你好开发者，现在加密文件夹只要输一次正确的密码，在不改文件夹密码的情况下就会一直有效，在后台能不能增加cookie密码失效时间选项，比如超过24H需要再次输入正确的密码进行验证。

**Comment by xhofe** [comment]
我想不到这样做的意义。

**Reply by ErDongChen1973** [reply]
我的现象是:浏览器第一次登进到某个文件夹后（我现在是根文件夹），即便我后台修改了密码，我打开浏览器还是能进去(不提示重新输入密码），除非清理了浏览器的cookie ，或者换成其他之前没登录过的浏览器 (这是个bug么？)

**Reply by xhofe** [reply]
有没有退出登录，登录之后是无视密码的。

**Reply by ErDongChen1973** [reply]
> 有没有退出登录，登录之后是无视密码的。

原来是我因为登录过管理员(所以一直能打开文件夹），不好意思，是我的问题
